imap_processing/ena_maps/utils/corrections.py

```diff
@@ -289,3 +298,241 @@ class or child classes.
             )
 
         return corrected_flux, corrected_flux_stat_unc
+
+
+def _add_spacecraft_velocity_to_pset(pset: LoHiBasePointingSet, et: float) -> None:
+    """
+    Calculate and add spacecraft velocity data to pointing set.
+
+    Parameters
+    ----------
+    pset : LoHiBasePointingSet
+        Pointing set object to be updated.
+    et : float
+        Ephemeris time in seconds past J2000.
```

Comment:
Can't this be obtained from the pset["epoch"] rather than having it as another parameter?